app/controllers/response_sets_controller.rb

```diff
@@ -46,6 +47,22 @@ def create
     end
   end
 
+  def transfer_table
+    @response_set.coded = @parent_set.coded
+    @response_set.oid = @parent_set.oid
```

Comment:
Oid should not be carried over, this is a different response set than that of the parent